ui/raidboss/data/triggers/e3s.js

```diff
@@ -115,6 +117,7 @@
         if (data.role == 'healer') {
           return {
             en: 'Tank Busters',
+            fr: 'Tankbursters',
```

Comment:
Typo?

Reply:
I don't know, I've checked other fights and sometimes it's 'Tankburster', sometimes 'Tank burster', not very consistant.

Do you still want me do update it ?

Reply:
Oh, I meant `Tankbuster` vs `Tankburster`.  I don't see the latter with the extra `r` anywhere else, so was curious if that was intended.

Reply:
Oh yeah, mb, gonna fix it

Reply:
Should be good now